### ISSUE TITLE
aws_cognito_user_group passes group description in AWS request as role_arn property

### ISSUE THREAD

**Issue by npellegrin** [terraform-providers/terraform-provider-aws#4236]
Hello.

I am facing a very strange behavior, I believe that it is a Terraform provider bug, but feel free to close the ticket if not :)

### Terraform Version
Terraform v0.11.6
+ provider.aws v1.14.1
+ provider.external v1.0.0
+ provider.local v1.1.0
+ provider.null v1.0.0

### Affected Resource(s)
- aws_cognito_user_group

### Terraform Configuration Files
```hcl
resource "aws_cognito_user_group" "team" {
  name        = "team"
  description = Team group (managed by Terraform)"

  user_pool_id = "${aws_cognito_user_pool.api_users.id}"
  role_arn     = "${aws_iam_role.opteam_role.arn}"
}
```

### Debug Output
https://gist.github.com/npellegrin/72b9d54bc494d95e712df5ab4d2e5061
Error message is: `Invalid ARN format Team group (managed by Terraform)`

### Expected Behavior
The group should have been modified with the correct role ARN.

### Actual Behavior
It seems that in the request performed to Amazon, Terraform sends the group description as role_arn (as seen in the error message) preventing to update the resource.

### Steps to Reproduce
`terraform apply`

**Comment by bflad** [comment]
Hi @npellegrin 👋  Thanks for reporting this and sorry you ran into the unexpected behavior.

It does indeed look like there is a bug with updating the `role_arn` with the `aws_cognito_user_group` resource:

https://github.com/terraform-providers/terraform-provider-aws/blob/29a75f23e19a1047fed36345c0221657cc06e49b/aws/resource_aws_cognito_user_group.go#L132-L134

Submitting a bugfix PR shortly.

**Comment by bflad** [comment]
Bugfix PR submitted: #4237

**Comment by npellegrin** [comment]
Thank you for the fix :)

**Comment by bflad** [comment]
This has been released in [version 1.15.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1150-april-18-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
